remotetests/remotetest.js

```diff
@@ -644,6 +644,57 @@ describe('Remote Tests', function() {
     });
   });
 
+  it('Check build logs from deployed URI', function(done) {
+    var opts = baseOpts();
+    opts['hosted-runtime'] = true;
+    opts.api = HOSTED_FUNCTIONS_PROXY_NAME;
+
+    var logStream = new stream.PassThrough();
+    logStream.setEncoding('utf8');
+    opts.stream = logStream;
+    apigeetool.getLogs(opts, function(err) {
+      assert(!err);
+
+      var allLogs = '';
+      logStream.on('data', function(chunk) {
+        allLogs += chunk;
+      });
+      logStream.on('end', function() {
+        try {
+          done();
```

Comment:
does `done()` throw? I thought it was just `mocha`'s mechanism for indicating an async test was done + passing errors up

Reply:
`done()` doesn't throw. You're correct on that it's used to tell when a test is done. There should probably be an `assert` in the try block. However there won't be any runtime logs to check and the above `assert(!err)` feels sufficient for the test.